posthog/models/event_buffer.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class EventBuffer(models.Model):
+    event: models.JSONField = models.JSONField(null=True, blank=True)
```

Comment:
Nit: `event` is usually a string for us, mixing data types is half of our typical pain. `event_data` would have been a better name.